FreeCite.js

```diff
@@ -0,0 +1,76 @@
+{
+	"translatorID": "7bbef39f-8bb9-44d7-826f-47ce75eb15ae",
+	"label": "FreeCite",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://freecite\\.library\\.brown\\.edu/citations/create",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2015-03-15 22:54:52"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2015 Philipp Zumstein
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (doc.getElementsByTagName("code").length) {
+		return "multiple";
+	}
+}
+
+function doWeb(doc, url) {
+	var texts = [], text = "";
+	var codes = doc.getElementsByTagName("code");
+	for(var i = 0; i < codes.length; i++) {
+		text = codes[i].textContent;
+		text.replace(/</g,"&lt;").replace(/>/g,"&gt;");
+		texts.push(text);
+	}
+	
+	var translator = Zotero.loadTranslator('import');
+	translator.setTranslator('24d9f058-3eb3-4d70-b78f-1ba1aef2128d');//CTX
+	translator.setString(texts);
```

Comment:
This can't be an array (though it may currently work by some miracle). Best choice would probably be to concatenate all of the `ctx:context-object` tags under a single `ctx:context-objects` and pass it as a single string to the import translator. Alternatively, you would have to call the import translator several times (I don't think we will have to resort to this)